src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/target.py

```diff
@@ -149,6 +157,8 @@ def fetch_accounts_for_target(self):
                 # in the correct way, starting with: 0o.
                 str(oct(int(self.path))).replace('o', ''),
             )
+        if str(self.path).startswith('//') and AWS_ACCOUNT_ID_REGEX.match(str(self.path[2:])):
```

Comment:
Could you change this to test that it starts with "ext-account-id://"
This would allow adding other types in the future too.

Reply:
I have added this. It now start with "ext-account-id://". Also external accounts are specified as "ext-..." in the CodePipeline.